src/test/java/javaserver/handlers/DirectoryHandlerTest.java

```diff
@@ -7,45 +7,42 @@
 
 public class DirectoryHandlerTest {
 
-    Request request = new Request();
-    Response response;
+    private static final String TEST_FILE_NAME = "test";
+    private static final String TEST_URI = "/test";
+    private static final String EXPECTED = String.format(DirectoryHandler.HTMLFORMAT, TEST_URI, TEST_FILE_NAME);
+
+    private Request request = new Request();
+    private Response response;
 
     @Test
-    public void createDirectoryListBody_GivenAnEmptyDirectory_ReturnsEmptyString() {
-        response = new MockWithEmptyDirectory().handleRequest(request);
+    public void givenNoFilesReturnsAnEmptyBody() {
```

Comment:
Just a nitpick but it might be better to switch to the order around like this:
`returnsAnEmptyBodyGivenNoFiles`.

Starting with "given" sounds like it should be a private function that a test uses